fusionfrontend/src/app/dashboards/components/content/maintenance-list/maintenance-list.component.ts

```diff
@@ -13,26 +13,29 @@
  * under the License.
  */
 
-import { Component, OnInit, Input, OnChanges } from '@angular/core';
-import { FactoryAssetDetailsWithFields, DashboardFilterModalType } from 'src/app/store/factory-asset-details/factory-asset-details.model';
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+import {
+  DashboardFilterModalType,
+  FactoryAssetDetailsWithFields
+} from 'src/app/store/factory-asset-details/factory-asset-details.model';
 import { faFilter, faSearch } from '@fortawesome/free-solid-svg-icons';
 import { AssetType } from 'src/app/store/asset-type/asset-type.model';
 import { FactorySite } from 'src/app/store/factory-site/factory-site.model';
 import { Company } from 'src/app/store/company/company.model';
 import { SelectItem } from 'primeng/api';
 
-
 interface ActiveFilter {
   filterAttribute: SelectItem;
 }
 
+export enum MaintenanceState { CRITICAL, MEDIUMTERM, LONGTERM}
+
 const CRITICAL_MAINTENANCE_VALUE = 375;
```

Comment:
can be set readonly rather than const for properties. const ist not immutability